### PR TITLE
Last update TiddlyStow-ZH.html

### PR DESCRIPTION
**Pull request by Zacharia2**

### CODE REVIEW

**Comment by Zacharia2** [comment]
@btheado hi，This is probably the last pr, if you see it and think it's okay, please merge it. thank you.

File: TiddlyStow-ZH.html [hunk 1]
```diff
@@ -51,39 +51,44 @@
...
           <li>
             并不是所有的浏览器都 <a href="https://caniuse.com/native-filesystem-api">支持本地文件特性</a>.
             <span id="support-message"><b>好消息！你的浏览器支持它。</b></span>
           </li>
+          <li>可用的桌面浏览器：Edge/Chrome版本大于86，Safari版本大于15.2，Opera版本大于72。</li>
+          <li>创建的太微文档可以使用TidGi的导入Wiki.HTML功能导入到TidGi（太记）中使用。</li>
+          <li>你也可以把创建的tiddlywiki.html文档放入安卓的<a href="https://github.com/donmor/Tiddloid/releases">tiddloid</a>中使用。</li>
           <li>仅适用于单个TiddlyWiki文件实例。</li>
-          <li>不喜欢在浏览器存储中保存最近的文件列表？点击 <a href="https://twpub-book.vercel.app/tiddlystow.html#norecent">这里</a> 并刷新页面。</li>
+          <li>不喜欢在浏览器存储中保存最近的文件列表？点击 <a href="tiddlystow.html#norecent">这里</a> 并刷新页面。</li>
```
Comment (btheado) [inline TiddlyStow-ZH.html]: Did you mean this?

```suggestion
          <li>不喜欢在浏览器存储中保存最近的文件列表？点击 <a href="TiddlyStow-ZH.html#norecent">这里</a> 并刷新页面。</li>
```

Comment (Zacharia2) [reply TiddlyStow-ZH.html]: yes，you are right！

Comment (Zacharia2) [reply TiddlyStow-ZH.html]: I've solved it！

**Comment by btheado** [comment]
Thanks, @Zacharia2

**Comment by Zacharia2** [comment]
@btheado You're welcome

**Pull request merged**